flow/connectors/bigquery/bigquery.go

```diff
@@ -217,7 +217,7 @@ func (c *BigQueryConnector) ReplayTableSchemaDeltas(flowJobName string,
 	schemaDeltas []*protos.TableSchemaDelta) error {
 	for _, schemaDelta := range schemaDeltas {
 		if schemaDelta == nil || len(schemaDelta.AddedColumns) == 0 {
-			return nil
+			continue
```

Comment:
It'd be useful to explain why `return nil` becomes `continue` in PR description

Reply:
added in the description, it is a bugfix